namespaces/mson-namespace.md

```diff
@@ -289,13 +277,10 @@ Enumeration type. Exclusive list of possible elements. The elements in content's
                     "element": "option",
                     "content": [
                         {
-                            "element": "member",
-                            "content": {
-                                "key": {
-                                    "element": "string",
-                                    "content": "state"
-                                }
-                            }
+                            "element": "string",
+							"meta": {
+								"name": "state"
+							}
```

Comment:
Strange. That was some tabs / space mismatch – looked OK in my editor. Fixing it right now